integrations/dd-sdk-android-compose/src/test/kotlin/ComposeActionTrackingStrategyTest.kt

```diff
@@ -34,38 +46,302 @@ class ComposeActionTrackingStrategyTest {
     private lateinit var testedComposeActionTrackingStrategy: ComposeActionTrackingStrategy
 
     @Mock
-    private lateinit var mockDecorView: View
+    private lateinit var mockView: View
+
+    @Mock
+    private lateinit var mockAndroidComposeView: AndroidComposeView
+
+    @Mock
+    private lateinit var mockLayoutNodeUtils: LayoutNodeUtils
 
     @BeforeEach
     fun `set up`() {
-        testedComposeActionTrackingStrategy = ComposeActionTrackingStrategy()
+        testedComposeActionTrackingStrategy = ComposeActionTrackingStrategy(
+            layoutNodeUtils = mockLayoutNodeUtils
+        )
     }
 
-    // TODO RUM-9298: Fix the test after implementing Compose action tracking strategy.
     @Test
-    fun `M return null W ComposeActionTrackingStrategy is not implemented yet {tap}`(forge: Forge) {
+    fun `M return null W find tap target in decorView {not a compose owner}`(forge: Forge) {
         // Given
-        val x = forge.aFloat()
-        val y = forge.aFloat()
+        val x = forge.mockSmallFloat()
+        val y = forge.mockSmallFloat()
 
         // When
-        val result = testedComposeActionTrackingStrategy.findTargetForTap(mockDecorView, x, y)
+        val result = testedComposeActionTrackingStrategy.findTargetForTap(mockView, x, y)
 
         // Then
         assertThat(result).isNull()
     }
 
-    // TODO RUM-9298: Fix the test after implementing Compose action tracking strategy.
     @Test
-    fun `M return null W ComposeActionTrackingStrategy is not implemented yet {scroll}`(forge: Forge) {
+    fun `M return null W find scroll target in decorView {not a compose owner}`(forge: Forge) {
         // Given
-        val x = forge.aFloat()
-        val y = forge.aFloat()
+        val x = forge.mockSmallFloat()
+        val y = forge.mockSmallFloat()
 
         // When
-        val result = testedComposeActionTrackingStrategy.findTargetForScroll(mockDecorView, x, y)
+        val result = testedComposeActionTrackingStrategy.findTargetForScroll(mockView, x, y)
 
         // Then
         assertThat(result).isNull()
     }
+
+    /**
+     *           0
+     *         /   \
+     *        1(c)  2
+     *      /     \
+     *     3(c)   4
+     */
+    @Test
+    fun `M return lowest level clickable node W findTargetForTap`(forge: Forge) {
```

Comment:
minor: let's use `bottom-most` instead of `lowest level`, seems like it is the opposite of the quite common `top-most`.